src/debug/webgl-debug.js

```diff
@@ -92,31 +92,31 @@ var glValidEnumContexts = {
   'getTexParameter': {2: { 0:true, 1:true }},
   'texParameterf': {3: { 0:true, 1:true }},
   'texParameteri': {3: { 0:true, 1:true, 2:true }},
-  'texImage2D': {
-     9: { 0:true, 2:true, 6:true, 7:true },
-     6: { 0:true, 2:true, 3:true, 4:true }
-  },
-  'texSubImage2D': {
-    9: { 0:true, 6:true, 7:true },
-    7: { 0:true, 4:true, 5:true }
-  },
+  //'texImage2D': {
+  //   9: { 0:true, 2:true, 6:true, 7:true },
+  //   6: { 0:true, 2:true, 3:true, 4:true }
+  //},
+  //'texSubImage2D': {
+  //  9: { 0:true, 6:true, 7:true },
+  //  7: { 0:true, 4:true, 5:true }
+  //},
```

Comment:
Concretely: I suggest adding a comment here like "texImage2D and texSubImage2D are defined below with WebGL 2.0 changes", and removing the commented-out code.